code/components/loading-screens-five/src/LoadingScreens.cpp

```diff
@@ -776,7 +776,7 @@ static InitFunction initFunction([] ()
 				}
 			}
 
-			uint64_t loadTimingIdentity = HashString(name) | (16 << 48);
+			uint64_t loadTimingIdentity = HashString(name) | (uint64_t(16) << 48);
```

Comment:
Good point. Default MSVC compiler warning verbosity makes it hard to catch some legitimate warnings.